spec/plans/report.fmf

```diff
@@ -87,6 +87,11 @@ description:
 
         __ https://github.com/RedHatQE/pylero
 
+        For Polarion report to export correctly you need to use password authentication,
```

Comment:
I'd recommend using either `note` or `warning` adminition, similar to `note` below. It will make the note stand up as something worth noticing.

Reply:
note seems more correct, thanks